src/config.js

```diff
@@ -46,12 +55,12 @@ const conf = convict({
   db: {
     host: {
       doc: 'Hostname where PostgreSQL is running',
-      format: 'url',
+      format: '*',
       default: 'localhost'
     },
     port: {
       doc: 'Port where PostgreSQL is running',
-      format: 'port',
+      format: '*',
```

Comment:
these changes are needed for me because I connect to psql through a unix socket (which is quite cool because it matches psql user with your user :) ).